src/api/pages.js

```diff
@@ -5,6 +5,11 @@ var routes    = require('./routes');
 var getJSONP  = require('../lib/getJSONP');
 var campaigns = require('./campaigns');
 
+function customJoin(ids, joinString) {
```

Comment:
Is this function used in two places now? Might be a good idea to put it in libs as a utility function so it's not duplicated.